<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startform</class>
 <widget class="QWidget" name="startform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>862</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RF Tool Config</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>601</width>
     <height>801</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainGridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="FOR_groupBox">
      <property name="title">
       <string>Формат данных</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>491</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout"/>
        </item>
        <item row="1" column="0">
         <widget class="QTabWidget" name="formTabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>1</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget_4">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>10</y>
              <width>261</width>
              <height>131</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Разрядность:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="FOR_binsizeComboBox">
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>16</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>32</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>64</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QComboBox" name="FOR_binendianComboBox">
               <item>
                <property name="text">
                 <string>little-endian</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>big-endian</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QComboBox" name="FOR_binsignComboBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>signed</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>unsigned</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="FOR_bincomplexCheckBox">
               <property name="text">
                <string>Комлексные (IQ)</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>65</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>0</y>
         <width>108</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="startRdPushButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startUpdPushButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="SRC_groupBox">
      <property name="title">
       <string>Источник данных</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>581</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Кол-во точек</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Fs:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="RD_cntSpinBox">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>10000</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="fsLineEdit">
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="fsComboBox">
            <item>
             <property name="text">
              <string>Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MHz</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="pow2ComboBox"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="SRC_onefileRadioButton">
            <property name="text">
             <string>Один файл - один канал</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="SRC_onefilechanRadioButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Один файл - несколько каналов</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="SRC_filesRadioButton">
            <property name="text">
             <string>Несколько файлов (каналов)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="SRC_sharedRadioButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Shared memory</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTabWidget" name="SRC_tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>1</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>21</x>
              <y>39</y>
              <width>441</width>
              <height>41</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Путь к файлу:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="DropLineEdit" name="SRC_onefileSelectLineEdit" native="true"/>
             </item>
             <item>
              <widget class="QPushButton" name="SRC_onefileselectFilePushButton">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_10">
           <attribute name="title">
            <string>2</string>
           </attribute>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>11</x>
              <y>19</y>
              <width>73</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Путь к файлу:</string>
            </property>
           </widget>
           <widget class="QPushButton" name="SRC_onefilechanselectFilePushButton">
            <property name="geometry">
             <rect>
              <x>375</x>
              <y>19</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="SRC_onefilechanLineEdit">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>20</y>
              <width>279</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>H:\RF_TOOL\other\gen.pcm</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>60</y>
              <width>191</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Сколько  точек относится к каналу</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="SRC_chanlenSpinBox">
            <property name="geometry">
             <rect>
              <x>220</x>
              <y>60</y>
              <width>42</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QComboBox" name="SRC_channComboBox">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>100</y>
              <width>69</width>
              <height>22</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>100</y>
              <width>121</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Количество каналов</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>3</string>
           </attribute>
           <widget class="QComboBox" name="SRC_chanNcomboBox">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>20</y>
              <width>69</width>
              <height>22</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>20</y>
              <width>121</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Количество каналов</string>
            </property>
           </widget>
           <widget class="DropTableWidget" name="SRC_chanTableWidget" native="true">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>60</y>
              <width>431</width>
              <height>101</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>4</string>
           </attribute>
           <widget class="DropTableWidget" name="SRC_sharedTableWidget" native="true">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>60</y>
              <width>431</width>
              <height>91</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>20</y>
              <width>121</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Количество каналов</string>
            </property>
           </widget>
           <widget class="QComboBox" name="SRC_sharedNcomboBox">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>20</y>
              <width>69</width>
              <height>22</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropLineEdit</class>
   <extends>QWidget</extends>
   <header>dropLineEdit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DropTableWidget</class>
   <extends>QWidget</extends>
   <header>dropTableWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
